src/vtrasterizer/TextRenderer.cpp

```diff
@@ -837,7 +837,13 @@ auto TextRenderer::createRasterizedGlyph(atlas::TileLocation tileLocation,
     }
 
     // y-position relative to cell-bottom of glyphs top.
-    auto const yMax = _gridMetrics.baseline + glyph.position.y;
+    auto yMax = _gridMetrics.baseline + glyph.position.y;
+
+    if(yMax < 0)
+    {
+        RasterizerLog()("Encountered glyph with inverted direction, swaping to normal");
+        yMax *= -1;
+    }
```

Comment:
```suggestion
    if (yMax < 0)
    {
        RasterizerLog()("Encountered glyph with inverted direction, swaping to normal");
        yMax = std::abs(yMax);
    }
```
While I am perfectly fine with inverting yMax by multiplying with `-1` I feel that using `|x|` instead is more readable, `abs(x)` I mean :)

Reply:
Also, as you also said this is a hack, I am not 100% certain that this is how it is supposed to be fixed. I mean, if it works, i'm happy, actually. But I didn't have the time yet to actually understand this change more deeply.

Reply:
NB: that is probably worth mentioning in the changelog section in `metainfo.xml`. Some user-facing text describing this fix.